CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## [4.8.1]
 
+#### Added
+- Update to okhttp3 version 5.1.0 [SDKS-4214]
```

Comment:
We recently got feedback about the lack of details on the CHANGELOG. So, it could be more detailed. I recommend adding specific details about the migration from OkHttp 4.x to 5.x and comment on any potential impacts or not impact at all.